README.rst

```diff
@@ -137,6 +137,11 @@ Active contributors
 
 .. _`Nicola Lambri`: https://github.com/nlambriICH
 
+* `Cody Crewson`_
+    * `Saskatchewan Cancer Agency`_, Canada
+
+.. _`Cody Crewson`: https://github.com/crcrewso
+
```

Comment:
If your organisation would like to have their logo on PyMedPhys, they may do this, given you are a contributor. But that is of course up to them.

Reply:
I'm not sure about the logo, we're going through rebranding. I will add the website though